BasicSteps/SweepLoopRange.cs

```diff
@@ -185,8 +186,15 @@ public SweepLoopRange()
             Rules.Add(() => ((SweepBehavior != SweepBehavior.Exponential)) || (Math.Sign(SweepStop) == Math.Sign(SweepStart)), "Sweep start and end value must have the same sign.", "SweepEnd", "SweepStart");
 
             ChildTestSteps.ChildStepsChanged += childStepsChanged;
+            PropertyChanged += SweepLoopRange_PropertyChanged;
         }
-        
+
+        void SweepLoopRange_PropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == nameof(ChildTestSteps))
+                ChildTestSteps.ChildStepsChanged += childStepsChanged;
```

Comment:
Can we remove the eventhandler from the old childsteps instance here to avoid leaking that?

Reply:
This is not possible in a PropertyChanged event handler, but that should not be a problem, since 'this' no longer points to that instance of ChildTestSteps that event handler should be collectable by GC.

Reply:
You are right. I just had it turned around in my head.